UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/Services/BuilderService.cs

```diff
@@ -301,11 +303,20 @@ private void LogProductVersions(ProductVersionsRequest productVersionsRequest, s
             }
         }
 
-        private async Task PerformAncillaryFilesOperationsAsync(string batchId, string exchangeSetPath, string correlationId, string readMeSearchFilter)
+        private async Task PerformAncillaryFilesOperationsAsync(string batchId, ConfigQueueMessage configQueueMessage, string essFileDownloadPath)
         {
+            string exchangeSetPath = Path.Combine(homeDirectoryPath, batchId, fssApiConfig.Value.BespokeExchangeSetFileFolder);
             string exchangeSetRootPath = Path.Combine(exchangeSetPath, fssApiConfig.Value.EncRoot);
             string readMeFilePath = Path.Combine(exchangeSetRootPath, fssApiConfig.Value.ReadMeFileName);
-            await CreateReadMeFileAsync(batchId, correlationId, readMeSearchFilter, exchangeSetRootPath, readMeFilePath);
+            await CreateReadMeFileAsync(batchId, configQueueMessage.CorrelationId, configQueueMessage.ReadMeSearchFilter, exchangeSetRootPath, readMeFilePath);
```

Comment:
this method can be moved down above UpdateSerialFileAsync method. 
@chaital14503 pls make these changes in PBI - 140041 or 154171

Reply:
Noted